lms/static/sass/discussion/_layouts.scss

```diff
@@ -1,31 +1,43 @@
 // Layouts for discussion pages
+@import '../course/base/extends';
 
-.user-profile {
-    background-color: $sidebar-color;
+.discussion-user-profile-board {
 
-    .user-profile {
-        padding: $baseline;
-        min-height: 500px;
+    .page-title {
+        @include margin-left($baseline);
+        display: inline-block;
     }
 
-    .sidebar-username {
-        font-weight: 700;
-        font-size: $forum-large-font-size;
+    .discussion-profile-info  {
+        @include margin(0, $baseline / 4, 0, $baseline);
     }
 
-    .sidebar-user-roles {
-        margin-top: $baseline/2;
+    .discussion-profile-count {
+        margin-top: $baseline / 4;
+    }
+
+    .user-name {
+        @include margin(0, $baseline / 4, 0, $baseline);
+        font-size: $forum-x-large-font-size;
+    }
+
+    .user-roles {
+        font-size: $forum-small-font-size;
         font-style: italic;
-        font-size: $forum-base-font-size;
     }
 
-    .sidebar-threads-count {
-        margin-top: $baseline/2;
+    .discussion-user-threads {
+        margin: $baseline auto;
+        width: 97%;
```

Comment:
This is a little weird - is this so the edge of the thread list aligns with the user name? I would just go with a 100% width here and remove the padding from the user name (or go with 100% width here and apply the same amount of padding) - shaving 3% width off here won't keep the two in alignment at all screen sizes, as the padding above isn't percentage-based